docker/install/ubuntu_init_zephyr_project.sh

```diff
@@ -16,10 +16,35 @@
 # specific language governing permissions and limitations
 # under the License.
 
+#
+# Initialize Zephyr Project.
+#
+# Usage: ubuntu_init_zephyr_project.sh path branch [--commit hash]
+#   path is the installation path for the repository.
+#   branch is the zephyr branch.
+#   --commit is the commit hash number of zephyrproject repository. If not specified, it uses the latest commit.
+#
+
+set -x
+
 DOWNLOAD_DIR=$1
-ZEPHYR_BRANCH=$2
+shift
+ZEPHYR_BRANCH=$1
+shift
+
+commit_hash="0"
```

Comment:
slightly better test could be:
```
commit_hash=
# ...

if [ -n "${commit_hash}" ]; then
  # ...

```

this avoids the problem that a user could supply `--commit 0` and not receive an error